.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,13 @@
 <!--- Make sure to add a descriptive title in the field above! E.g. "Fixes the header title color on the homepage"  -->
 
+## Ember Times review template
+
+- [ ] Author's name at top and bottom
+- [ ] Put emoji in writeup title
```

Comment:
Before this TODO item, can we add another TODO item regarding adding a short blurb at the beginning?

Reply:
i think i got it where you want it!